.studio/golang

```diff
@@ -184,6 +184,11 @@ function go_update_component() {
 # Adding auto tab complete
 complete -F _component_auto_complete go_update_component
 
+document "kill_running_service" <<DOC
+  kill a running component. This can be used to restart a component
+
+  example kill_running_service ingest-service
```

Comment:
It'd be more clear what the command to run was if this was split onto multiple lines.
```suggestion
  Example:
      kill_running_service ingest-service
```

Reply:
I'm not sure if we have a convention for this.

Reply:
Thank you.

Reply:
I am not sure why but that broke the DCO check. :man_shrugging:

Reply:
That's an unfortunate side effect of the suggestion feature and DCO check not working well together https://chefio.slack.com/archives/C0808K15Y/p1561751386240400